workers/ohsome_quality_analyst/api/api.py

```diff
@@ -280,47 +280,47 @@ async def get_available_regions(asGeoJSON: bool = False):
         return response
 
 
-@app.get("/indicatorLayerCombinations")
-async def list_indicator_layer_combinations():
-    """List names of available indicator-layer-combinations."""
+@app.get("/indicators-layers")
```

Comment:
It's not intuitive to me what I would get from `/indicators-layers`.

Reply:
How about `indicator-layer-combinations`?

Reply:
Done